### ISSUE TITLE
NOD | Add VANotify credentials to staging and production

### ISSUE THREAD

**Issue by data-doge** [department-of-veterans-affairs/va.gov-team#76113]
## Value Statement

As the VA
I need VANotify keys to be added to VA.gov's staging and production environments
So that I can use VANotify to send notification emails to Veterans potentially impacted by the NOD infinite loop bug.

## Background Context

In https://github.com/department-of-veterans-affairs/va.gov-team/issues/74832, we set up a status callback url for notifications sent with VANotify, and verified it worked as expected in staging.  Now, we need to add VANotify's staging and production keys to VA.gov's staging and production environment.  They have already sent the keys (service uuids and api keys) to my (Eugene's) VA.gov email address.

N.B. we will have to save our api key in AWS parameter store with the following format: `benefits-decision-review-primary-key-SERVICE_UUID-API_KEY_HERE`.  See https://dsva.slack.com/archives/C010R6AUPHT/p1707844709201799?thread_ts=1707516849.079339&cid=C010R6AUPHT.

## Tasks

- [x] Decide what we want to call our api key.
  For consistency we will use the naming convention already in place for other VANotify API keys.
  `Settings.vanotify.services.benefits_decision_review.api_key`
  Staging key: `/dsva-vagov/vets-api/staging/vanotify/services/benefits_decision_review/api_key`
  Prod key: `/dsva-vagov/vets-api/prod/vanotify/services/benefits_decision_review/api_key`
- [x] Add api key to AWS parameter store, for staging, and prod.
- [x] Update vsp-infra-application-manifests to include a reference to the api key in staging and prod. See https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2668/files for guidance.
- [x] Create a test email template in VANotify.
- [x] Run the following code to send a test email with that template
  ```
  notify_client = VaNotify::Service.new(Settings.vanotify.services.benefits_decision_review.api_key)
  notify_client.send_email({ email_address: <email>, template_id: <template_id> })
  ```

## Acceptance Criteria
- [x] We can successfully send a test email in staging and prod, using VANotify code above.

## Definition of Ready
N/A. Ready now.

## Definition of Done
- [x] Acceptance criteria met.
- [x] We have a follow-up ticket, or plan, to set up and test whatever templates VA needs for the NOD infinite loop email blast.

**Comment by dfong-adh** [comment]
Created 3 PRs for the remaining secrets to be added to staging and production.

**Comment by HeatherWidmont** [comment]
@data-doge added the secrets, now waiting on those PRs to be merged

**Comment by saderagsdale** [comment]
@data-doge will ping platform today for merges.

**Comment by data-doge** [comment]
Works in staging and production. Closing.
